### PR TITLE
Added "path" field to ConsulRawClient

### PR DESCRIPTION
**Pull request by ianmcderp**
Added "path" field to `ConsulRawClient` to access Consul behind a proxy as mentiond in #77. I also implemented a `Builder` in `ConsulRawClient` to allow easier configuration of a client.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,11 +9,12 @@ repositories {
 
 dependencies {
 	compile "com.google.code.gson:gson:2.3"
-
+	compile "org.apache.commons:commons-lang3:3.0"
```
Comment (spencergibb) [inline build.gradle]: I'd say that adding a dependency just to check if a string is empty isn't a good idea.

Comment (ianmcderp) [inline build.gradle]: Good point - do you want me to add a similar method like `isNotBlank(path)` to the project's `Utils` class instead?

Comment (ianmcderp) [inline build.gradle]: Removed dependency from the project and replaced occurrence of `isNotBlank` with `not null + not empty` check.

**Comment by vgv** [comment]
Thank you!

**Pull request merged**